test/es6.js

```diff
@@ -1,16 +1,25 @@
 var esprima = require('esprima');
 
 function tryThis(str, feature) {
-    try {
-        /*jshint evil: true */
-        eval(str);
-    } catch (ex) {
-        console.error('ES6 feature [' + feature + '] is not available in this environment');
-        return false;
+    // We can test instrumentation of exports even if the environment doesn't support them. 
+    if (feature !== 'export') {
+        try {
+            /*jshint evil: true */
+            eval(str);
+        } catch (ex) {
+            console.error('ES6 feature [' + feature + '] is not available in this environment');
+            return false;
+        }
```

Comment:
Instead of making the environment detection "pass", could you rather not just remove the conditional check in `test-es6-export.js` that tries to do environment detection?